shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/builder/loader/ColumnMetaDataLoader.java

```diff
@@ -37,21 +38,21 @@
  */
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public final class ColumnMetaDataLoader {
-    
+
     private static final String COLUMN_NAME = "COLUMN_NAME";
-    
+
     private static final String DATA_TYPE = "DATA_TYPE";
-    
+
     private static final String TYPE_NAME = "TYPE_NAME";
-    
+
     private static final String TABLE_NAME = "TABLE_NAME";
-    
+
     /**
      * Load column meta data list.
-     * 
-     * @param connection connection
+     *
+     * @param connection       connection
```

Comment:
@chenxk Please refer the doc of [Code of Conduct](https://shardingsphere.apache.org/community/en/contribute/code-conduct/), and execute checkstyle first.